### PR TITLE
Done

### PR DESCRIPTION
**Pull request by MaksimusCRS**
[DEMO LINK](https://MaksimusCRS.github.io/react_clock/)

### CODE REVIEW

**Review by luckdenny — APPROVED** [review]

File: src/App.scss [hunk 1]
```diff
@@ -1 +1,6 @@
-// styles go here
+.App {
+  display: inline-block;
+  border: 1px solid red;
+  text-align: center;
+  padding:  10px;
+}
```
Comment (Elena-brv) [inline src/App.scss:6]: empty line

File: src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,59 @@
...
+interface State {
+  isClockVisible: boolean;
+  clockName: number;
+}
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+export class App extends React.Component<{}, State> {
+  state = {
```
Comment (Elena-brv) [inline src/App.tsx:12]:
```suggestion
  state: State = {
```

File: src/App.tsx [hunk 3]
```diff
@@ -1,24 +1,59 @@
...
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <button
+          type="button"
+          onClick={this.hideClock}
+        >
+          Hide Clock
+        </button>
+        {' '}
```
Comment (Elena-brv) [inline src/App.tsx:41]: why do you need it?

**Review by Elena-brv — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 3 above)
Comment (MaksimusCRS) [reply src/App.tsx:41]: it was simple way to add spaces without styles)

**Review by Elena-brv — APPROVED** [review]
do not leave empty files in project 
image

**Pull request closed without merging**